website/client/src/components/userMenu/profile.vue

```diff
@@ -899,6 +905,7 @@ export default {
     selectPage (page) {
       this.selectedPage = page || 'profile';
       window.history.replaceState(null, null, '');
+      this.oldTitle = this.$store.state.title;
```

Comment:
I haven't tested this @aevix but my guess is that `selectPage` is called not only the first time the modal is opened but also when switching, for example, from Stats to Achievements, in which case it would store the wrong title. I think this should be called in `mounted` so it's called just once